client-js/gulpfile.js

```diff
@@ -148,6 +148,8 @@ gulp.task('images', function () {
 // Copy All Files At The Root Level (app)
 gulp.task('copy', function () {
   var app = gulp.src([
+    '.tmp/**/*.js',
+    '.tmp/**/*.js.map',
```

Comment:
Now I'm paranoid about things hapening to gulp. What's going on here? What is gulp copy supposed to do?